Azure/sonic-buildimage
# [Mellanox] Optimize SFP Platform API implementation

**- Why I did it**

Each SFP object inside Chassis will open an SDK client, this is not necessary and SDK client can be shared between SFP objects.

**- How I did it**

Open one SDK client in Chassis init phase, pass this SDK client handle to all the SFP objects when init them.
Remove SDK client open/close code from the SFP object init/delete process.

**- How to verify it**

Verify current pmon daemons work well with these change.
Call related platform API and test the result.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
